nixio/tag.py

```diff
@@ -113,14 +119,44 @@ def create_feature(self, data, link_type):
         feat = Feature.create_new(self.file, self, features, data, link_type)
         return feat
 
+    def _calc_data_slices(self, data, position, extent, stop_rule):
+        refslice = list()
+        if not self.units:
+            units = [None]*len(data.dimensions)
+        else:
+            units = self.units
+
+        for idx, dim in enumerate(data.dimensions):
+            if idx < len(position):
+                pos = position[idx]
+                start = self._pos_to_idx(pos, units[idx], dim, SliceMode.Inclusive)
```

Comment:
I think this is not correct (since it is rather convoluted by nature, I might be wrong):
Consider the case of a `SampledDimension` with `sampling_interval = 1` and `offset = 0`. The Tag's position is 0.5.
This will call `index = dim.index_of(pos * scaling, stop_rule == SliceMode.Inclusive)` and eventually `SampledDimension::index_of(..., leq=True)`. This will return (since no exact match) the index 0. For a slice with start and stop the values returned now range from positions < start to somewhere else. For the end positions the calculations appear correct.

Reply:
Yes it will return index 0.  Isn't that correct?

Reply:
> Yes it will return index 0. Isn't that correct?

Right, I read the doc wrong.  Fixing now.